week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -38,9 +53,30 @@ Write a function that:
 */
 
 function formatPercentage(arr) {
-  
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] > 100) {
+      arr[i] = 100;
+    }
+    arr[i] = Math.round(arr[i] * 100) / 100;
```

Comment:
Very nice, clean code - you can use the `.toFixed()` method here to go straight from a number to a string with fixed decimal points